test/cds.ql.test.js

```diff
@@ -309,7 +309,7 @@ describe('cds.ql → cqn', () => {
                 'and',
                 { ref: ['args'] },
                 'in',
-                { val: args },
+                { list: [{ val: 'foo' }, { val: 'bar' }, { val: 3 }] },
```

Comment:
Shouldn't that be this?:
```suggestion
                { list: [{ val: 'foo' }, { val: "'bar'" }, { val: 3 }] },
```

Reply:
> Shouldn't that be this?:

I found that we trim single quotes in case fo Fluid API. Should it be in sync?

Reply:
I don't think so → we should also double check why we trim quotes in fluent API; sounds wrong

Reply:
Updated. Also, provided a bit more info per email.